File: Meta/META-INF/resources/lark/org/bark/test/Result.java
package org.bark.test;

@RemotingBean
public class Result {
  private String _content;

  public String  getContent() {
    return this._content;
  }
  public void setContent(String value) {
    this._content = value;
  }
  static {
    org.bark.remoting.SerializerFactory.getInstance().register(Result.class, new org.bark.remoting.Serializer(){
      @Override
      public void writeObject(javax.json.JsonObjectBuilder builder, org.bark.remoting.ReferenceProcessor handler, Object value) {
        Result __o = (Result)value;
        builder = __o._content == null ? builder.addNull("_content") : builder.add("_content", __o._content);
      }
    });
    org.bark.remoting.DeserializerFactory.getInstance().register(Result.class, new org.bark.remoting.Deserializer(){
      @SuppressWarnings({ "unchecked", "rawtypes" })
      @Override
      public Object readObject(javax.json.JsonObject jsonObj, Object[] handlers, Object obj) throws Exception {
        Result __o = (Result)obj;
        __o._content = jsonObj.get("_content") == javax.json.JsonValue.NULL ? null : jsonObj.getString("_content");
        return obj;
      }
    });
  }
}

File: Meta/META-INF/resources/lark/org/bark/test/Result.js
(function(){ 
  function Result() {    
    this._content = null;
  }
  Result.prototype.__proto__ = Object.prototype;
  __cache["org.bark.test.Result"] = Result;
  Object.defineProperty(Result.prototype, "content", {
    get : function() {
      return this._content;
    }, 
    set : function(value) {
      this._content = value;
    }
  });
  Result.prototype.__readObject = function(json, handlers, obj) {
    var __propVal = null;
    obj["_content"] = json["_content"];
  };
  Result.prototype.__writeObject = function(obj, handlers) {
    var __r = {"__clazz" : "org.bark.test.Result"};
    var __propVal = null;
    __r["_content"] = obj["_content"];
    return __r;
  };
  Result.prototype.__class = new (__lc('java.lang.Class'))("org.bark.test.Result", Result, Object.prototype.__class, [], 1);
  return  Result;
})();